src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -58,6 +59,9 @@ public void RunApplication(Application app)
         {
             try
             {
+                Dispatcher.CurrentDispatcher.UnhandledException += CurrentDispatcher_UnhandledException;
```

Comment:
can we unsubscribe these in shutdown, I'm not sure where this method is called from but it is public, perhaps it's called multiple times in the same process by some clients?